src/test/resources/MakeAppointment.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Make Appointment
+
+  An a user
+  I want to sign in
+  So I can use the app
+  After that I fill fields to make my appointment
+
+  @automated
+  Scenario Outline: User can make appointment with valid value
+    Given user is in Katalon login page
+    When user sets username as "John Doe" and password as "ThisIsNotAPassword"
+    And user clicks on login button
+    Then user sees appointment page
+    When user select facility as "<facilityBox>"
+    And user "<action>" apply
+    And user chooses "<program>"
+    And user decides "<date>"
+    And user adds "<comment>"
+    And user clicks on Book appointment button
+    Then user created appointment successfully
+    And user sees "<facilityBox>" as input
+    And user observes "<action>" as input
+    And user can see "<program>" as input
+    And user can observe "<date>" as input
+    And user notices "<comment>" as input
```

Comment:
tương tự như trên c cũng viết rõ ra thì sẽ hợp lý hơn 
`Then user sees facility is displayed as "<facilityBox>"`